dashboard/src/components/cfp-public/ActionsForm.vue

```
@@ -0,0 +1,83 @@
+</template>
+<script setup>
+import RenderField from '@/components/form/RenderField.vue'
+import ReferencesComponent from '@/components/cfp-public/ReferencesComponent.vue'
+import { toast } from 'vue-sonner'
+import { ref } from 'vue'
+import { Button, Dialog, createDocumentResource } from 'frappe-ui'
+
+const cfpid = defineModel('cfpid', {
+  type: String,
+  required: true,
+})
```

ESLint

[error] Duplicate key 'cfpid'. May cause name collision in script or template tag.

(vue/no-dupe-keys)